stories/DefaultForm.jsx

```diff
@@ -200,12 +200,19 @@ export default class DefaultForm extends Component {
             </div>
 
             <MyInput
-              name="foo"
+              name="initialValue"
               initialValue="Pooper"
               placeholder="My placeholder"
               disabled={ isDisabled }
               required />
 
```

Comment:
Please add the test instead of a story.

You may add it under Field.spec.js, if any. Thanks.